model/activity.go

```diff
@@ -1,21 +1,77 @@
 package model
 
 import (
```

Comment:
group packages by the type (core libraries, 3rd party libraries, and our own libraries) and separate groups by the new line